instance/static/html/instance/appserver.html

```diff
@@ -0,0 +1,57 @@
+<h4>{{appserver.name}}</h4>
+
+
+<div ng-if="!appserver">
+  <p>Loading AppServer details...</p>
+</div>
+
+<accordion close-others="false" ng-if="appserver">
+
+  <accordion-group heading="Status" is-open="true">
+    <table>
+      <tbody>
+        <tr><th>Status</th><td><strong>{{appserver.status_name}}</strong> ({{appserver.status_description}})</td></tr>
+        <tr><th>Active</th><td><span ng-if="is_active">Yes</span><span ng-if="!is_active">No</span></td></tr>
+        <tr><th>Created</th><td>{{appserver.created | date:'yyyy-MM-dd HH:mm:ssZ' }}</td></tr>
+        <tr><th>VM</th><td>{{appserver.server.name}}</td></tr>
+        <tr><th>VM Status</th><td>{{appserver.server.status_name}} ({{appserver.server.status_description}})</td></tr>
+        <tr><th>VM IP Address</th><td><a href="{{appserver.protocol}}://{{appserver.server.public_ip}}">{{appserver.server.public_ip}}</a></td></tr>
```

Comment:
@bradenmacdonald Nit: This would benefit from a few more newlines:, e.g.:

```html
        <tr>
          <th>Status</th>
          <td><strong>{{appserver.status_name}}</strong> ({{appserver.status_description}})</td>
        </tr>
        ...
```